include/modem/sms.h

```diff
@@ -196,11 +196,13 @@ void sms_unregister_listener(int handle);
  * in 3GPP TS 23.038 Section 4 and Section 6.2.
  * This function doesn't support sending of 8bit binary data messages or UCS2 encoded text.
  *
- * @param[in] number Recipient number.
+ * @param[in] number Recipient number in international format.
  * @param[in] text Text to be sent.
  *
  * @retval -EINVAL Invalid parameter.
- * @return 0 on success, otherwise error code.
+ * @return 0 on success, otherwise error code. A positive value is an AT error, where
+ *         the type can be resolved with @c nrf_modem_at_err_type and
+ *         the error value with @c nrf_modem_at_err.
```

Comment:
@peknis What's the format we should use to refer to a function in nrfxlib?

Reply:
Not sure, but I will ask my colleagues. In general for linking to functions in doxygen, see https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#references

Reply:
I've modified these now to have @ref instead of @c. I don't know if it should be @ref nrfxlib:nrf_modem_at_err or something similar but I'll wait what you can find out.

Reply:
`@ref nrf_modem_at_err` will cause documentation build to fail as the reference is not found. I'll try `@ref nrfxlib:nrf_modem_at_err`

Reply:
`@ref` doesn't work with or without `nrfxlib:` so I'll go back to `@c`

Reply:
I'll close this comment.
@peknis, please check the PR and approve if it's fine.